src/libs/scene_utils/README.md

```diff
@@ -4,62 +4,114 @@
 
-```
+```bash
 cd $GETTING_STARTED_DIR/src/libs/scene_utils
 npm install
 ```
 
+Set the following environment variables before running any sample script:
+
+```bash
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_SESSION_TOKEN
+AWS_REGION
+```
+
+You need to set your AWS account credentials and the region of your workspace for the AWS SDK used in these scripts. Even if you configured the variable `AWS_DEFAULT_REGION` you still need to set `AWS_REGION` because it is a key variable used by all clients in the AWS SDK.
+
+### Usage
+
+Run the following command in the directory of the sample script to see its usage.
+
+For example, to understand the usage of the Cookie Factory sample:
+
+```bash
+npx ts-node samples/cookie_factory_sample/sample.ts -h
+```
+
 # Samples:
 
 ## 1. COOKIE FACTORY `samples/cookie_factory_sample/`
 
+Create a Cookie Factory example scene in your workspace.
+
 Use the models in the `scenes` directory for the `--assetDirPath` input.
 
+### Prerequisites
+
+In order to run the Cookie Factory sample script you need to complete steps 4-6 to have entities and components with property data in your workspace.
+
 ### Example
 
-```
+```bash
 npx ts-node samples/cookie_factory_sample/sample.ts --workspaceId [WORKSPACE_ID] --sceneId [SCENE_ID] --assetDirPath $GETTING_STARTED_DIR/src/workspaces/cookiefactory/scenes/
 ```
 
 ### Output
 
 CookieFactory TwinMaker scene with Mixer nodes tied to data, Cookie Line nodes, and a WaterTank with a Motion Indicator widget.
 
-You will need to finish the Getting Started sample [setup](https://github.com/aws-samples/aws-iot-twinmaker-samples/blob/main/README.md) for the scene to be connected to data.
+You will need to finish the Getting Started sample [setup](https://github.com/aws-samples/aws-iot-twinmaker-samples/blob/main/README.md) for Mixers to be added with tags connected to data.
 
 ---
 
 ## 2. CESIUM `samples/cesium_sample/`
 
-Automates the Cesium Ion pipeline to export directly to TwinMaker.
+Automates the Cesium Ion pipeline to export directly to IoT TwinMaker.
 
 1. Upload your 3D asset to Cesium and wait for it to perform a tiling optimization.
-2. Download the tileset for the asset and upload it to your workspace's S3 bucket.
+2. Export the tileset directly to your workspace's S3 bucket.
 3. Create/edit a TwinMaker scene with the 3D tileset
 
+### Prerequisites
+
+1. Register Cesium and choose the appropriate [subscription plan](https://cesium.com/platform/cesium-ion/pricing/) based on your business requirement
+2. Create a Cesium asset token
+
+   a. Go to the [tokens page](https://cesium.com/ion/tokens)
+
+   b. Click “Create token”
+
+   c. Name it “TwinMaker token”
+
+   d. Toggle on the following permission scopes: 1. assets:list 2. assets:read 3. assets:write 4. exports:read 5. exports:write
+
+   e. Click “Create”
+
+   f. Copy and use this asset token for the `--cesiumAccessToken` parameter in the commands below
```

Comment:
how "confidential" is the Cesium token? Is it ok for it to be exposed in the command line as an argument or should it be via environment variables like the AWS credentials?

Reply:
If someone has access to a Cesium token then they will be able to perform any action the token allows in the customer's Cesium account. If it's more secure as an env var then I can enforce the use of an env var for this token.